examples/example-orca/src/main/java/io/grpc/examples/orca/CustomBackendMetricsClient.java

```diff
@@ -0,0 +1,115 @@
+import io.grpc.examples.helloworld.GreeterGrpc;
+import io.grpc.examples.helloworld.HelloReply;
+import io.grpc.examples.helloworld.HelloRequest;
+import io.grpc.internal.JsonParser;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * A simple xDS client that requests a greeting from {@link CustomBackendMetricsServer}.
+ * The client channel is configured to use an example load balancer policy
+ * {@link CustomBackendMetricsLoadBalancerProvider} which integrates with ORCA metrics reporting.
+ */
+public class CustomBackendMetricsClient {
+  private static final Logger logger = Logger.getLogger(CustomBackendMetricsClient.class.getName());
+
+  private final GreeterGrpc.GreeterBlockingStub blockingStub;
+
+  /** Construct client for accessing HelloWorld server using the existing channel. */
+  public CustomBackendMetricsClient(Channel channel) {
+    blockingStub = GreeterGrpc.newBlockingStub(channel);
+  }
+
+  /** Say hello to server. */
+  public void greet(String name) {
+    logger.info("Will try to greet " + name + " ...");
+    HelloRequest request = HelloRequest.newBuilder().setName(name).build();
+    HelloReply response;
+    try {
+      response = blockingStub.sayHello(request);
+    } catch (StatusRuntimeException e) {
+      logger.log(Level.WARNING, "RPC failed: {0}", e.getStatus());
+      return;
+    }
+    logger.info("Greeting: " + response.getMessage());
+  }
+
+  /**
+   * Greet server. If provided, the first element of {@code args} is the name to use in the
+   * greeting. The second argument is the target server.
+   */
+  public static void main(String[] args) throws Exception {
+    String user = "orca tester";
+    // The example defaults to the same behavior as the hello world example.
+    // To receive more periodic OOB metrics reports, use duration argument to a longer value.
+    String target = "localhost:50051";
+    long timeBeforeShutdown = 1500;
+    if (args.length > 0) {
+      if ("--help".equals(args[0])) {
+        System.err.println("Usage: [name [duration [target]]]");
+        System.err.println("");
+        System.err.println("  name    The name you wish to be greeted by. Defaults to " + user);
+        System.err.println("  duration  The time period in milliseconds that the server application " +
+            "wait until shutdown. Defaults to " + timeBeforeShutdown);
+        System.err.println("  target  The server to connect to. Defaults to " + target);
+        System.exit(1);
+      }
+      user = args[0];
+    }
+    if (args.length > 1) {
+      timeBeforeShutdown = Long.parseLong(args[1]);
+    }
+
+    if (args.length > 2) {
+      target = args[2];
+    }
+
+    LoadBalancerRegistry.getDefaultRegistry().register(
+        new CustomBackendMetricsLoadBalancerProvider());
+    ManagedChannel channel = ManagedChannelBuilder.forTarget(target)
+        .defaultServiceConfig(serviceConfig())
```

Comment:
FYI: `defaultLoadBalancingPolicy(EXAMPLE_LOAD_BALANCER)` would also work here. (This is unique to Java currently.)